### PR TITLE
Max Chapter2/Homework/3

### PR DESCRIPTION
**Pull request by HellHunterMax**
Restarted. changed everything so factories do all the work. tips are welcome!

### CODE REVIEW

**Comment by HellHunterMax** [comment]
i changed names from GetCpu to a Property Cpu, that's probably why the test are failing i changed them in my build.

File: Src/BootCamp.Chapter/Factory.cs [hunk 1]
```diff
@@ -5,40 +5,34 @@ namespace BootCamp.Chapter
...
-        public Ram AddRam(string brand)
-        {
-            PrintAddMsg("Ram", brand);
-            return new Ram(brand + "Ram");
-        }
-        private void PrintAddMsg(string component, string brand)
-        {
-            Console.WriteLine($"Adding {component} to {brand}PC.");
+            DesktopComputer PC = new DesktopComputer();
+            PC.Body = InstallBody();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Factory.cs]: Does InstallBody installs a body? No- it just creates it. You should pass a computer and then install the new piece to the passed computer. So please redo based on that.

Comment (HellHunterMax) [reply Src/BootCamp.Chapter/Factory.cs]: Oké i think i understand it now. please review again. small change.

**Comment by Almantask** [comment]
> nonono! I'll do it till perfection now! Anything else Sensei?

Nein!

**Pull request closed without merging**